Makefile

```diff
@@ -49,11 +49,18 @@ install-dev:
 install-build:
 	pip install -r requirements/build.txt
 
-## install-ingest-s3:       install requirements for the s3 connector
 .PHONY: install-ingest-s3
```

Comment:
The double `##` means that when the user types `make` those are the list of advertised targets. I think it is worth advertising all of the `install-ingest-` targets.

Reply:
Ok I'll put that back in!